cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2631,6 +2631,15 @@
   </property>
 
   <property>
+    <name>security.keytab.path</name>
+    <value></value>
+    <description>
+      Configuration for specifying the location of kerberos keytabs used for impersonation.
+      The location can contain ${name} which will be replaced by the user/owner of the entities name.
```

Comment:
Location of .... replaced by the short user name of the principal being impersonated.